app/controllers/api/v2/accounts_controller.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Api::V2::AccountsController < Api::V1::AccountsController
+  def block
+    BlockService.new.call(current_user.account, @account, stealth: !truthy_param?(:hard_block))
```

Comment:
Does this require a new API? It seems like you're essentially adding a new `stealth` param. Whether clients start using the new API or adding a new param to the old one, the result is the same.

Reply:
The idea was that the default with the new API would be a stealth block. But maybe that doesn't make much sense.

Reply:
Removed that endpoint, used `truthy_param?(stealth_block)` instead, meaning it's not the default in the API, but it's the default in the UI.